homeassistant/components/insteon/__init__.py

```diff
@@ -121,23 +126,37 @@ def set_default_port(schema: Dict) -> Dict:
     vol.Required(SRV_ALL_LINK_MODE): vol.In([SRV_CONTROLLER, SRV_RESPONDER]),
     })
 
+
 DEL_ALL_LINK_SCHEMA = vol.Schema({
     vol.Required(SRV_ALL_LINK_GROUP): vol.Range(min=0, max=255),
     })
 
+
 LOAD_ALDB_SCHEMA = vol.Schema({
     vol.Required(CONF_ENTITY_ID): cv.entity_id,
     vol.Optional(SRV_LOAD_DB_RELOAD, default='false'): cv.boolean,
     })
 
+
 PRINT_ALDB_SCHEMA = vol.Schema({
     vol.Required(CONF_ENTITY_ID): cv.entity_id,
     })
 
+
 X10_HOUSECODE_SCHEMA = vol.Schema({
     vol.Required(SRV_HOUSECODE): vol.In(HOUSECODES),
     })
 
+
+TRIGGER_SCENE_SCHEMA = vol.Schema({
+    vol.Required(SRV_ALL_LINK_GROUP): vol.Range(min=0, max=255)})
+
+
+LOAD_ALL_ALDB_SCHEMA = vol.Schema({
+    vol.Optional(SRV_LOAD_DB_RELOAD, default='false'): cv.boolean,
```

Comment:
FYI, I made the change but the original code does work since the helper method `boolean` does accept 'false` and False as the same value. Curious if there was a different reason for the recommendation.

Reply:
I didn't know. That's weird in my opinion. It's more clear to use a real boolean.